src/graph/UpdateEdgeExecutor.cpp

```diff
@@ -234,20 +234,26 @@ void UpdateEdgeExecutor::updateEdge(bool reversely) {
         for (auto& code : rpcResp.get_result().get_failed_codes()) {
             switch (code.get_code()) {
                 case nebula::storage::cpp2::ErrorCode::E_INVALID_FILTER:
-                      doError(Status::Error("Maybe invalid edge or property in WHEN clause!"));
-                      return;
+                    doError(Status::Error("Maybe invalid edge or property in WHEN clause!"));
+                    return;
                 case nebula::storage::cpp2::ErrorCode::E_INVALID_UPDATER:
-                      doError(Status::Error("Maybe invalid property in SET/YIELD clasue!"));
-                      return;
+                    doError(Status::Error("Maybe invalid property in SET/YIELD clasue!"));
+                    return;
+                case nebula::storage::cpp2::ErrorCode::E_FILTER_OUT:
+                    // Return ok when filter out without exception
+                    // so do nothing
+                    // https://github.com/vesoft-inc/nebula/issues/1888
+                    // TODO(shylock) maybe we need alert user execute ok but no data affect
```

Comment:
Call doFinish here.

Reply:
This need put columns to next executor such as yield if exist.

Reply:
Thanks.